tests/ci/fast_test_check.py

```diff
@@ -95,6 +96,8 @@ def process_results(result_folder):
 
     gh = Github(get_best_robot_token())
```

Comment:
We should add `per_page=100` here and in other places, otherwise, we could spend out all our requests' limits for nothing